src/tests/functional/shared_test_classes/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ addIeTarget(
                 funcTestUtils
                 ngraphFunctions
                 inference_engine_legacy
+                inference_engine_transformations
```

Comment:
the comment is the same as for other changed cmake file

Reply:
fixed. please check again. thanks a lot!